src/main/resources/bootstrap.yaml

```diff
@@ -5,13 +5,27 @@ spring:
       paths: /mnt/secrets/bulk-scan
       aliases:
         processor-POSTGRES-PASS: BULK_SCANNING_DB_PASSWORD
+        processor-POSTGRES-USER: BULK_SCANNING_DB_USER_NAME
+        processor-POSTGRES-DATABASE: BULK_SCANNING_DB_NAME
+        processor-POSTGRES-HOST: BULK_SCANNING_DB_HOST
+        processor-POSTGRES-PORT: BULK_SCANNING_DB_PORT
```

Comment:
Not sure why we need those when we already have it defined https://github.com/hmcts/bulk-scan-processor/blob/master/charts/bulk-scan-processor/values.yaml#L27-L30